addons/sale_management/models/sale_order_option.py

```diff
@@ -148,6 +148,8 @@ def add_option_to_order(self):
             raise UserError(_('You cannot add options to a confirmed order.'))
 
         values = self._get_values_to_add_to_order()
+        max_sequence = max(self.order_id.order_line.mapped('sequence') or [0])
+        values['sequence'] = max_sequence + 1
```

Comment:
Couldn't you move this to `_get_values_to_add_to_order`? It would look cleaner to me.
```suggestion
        values['sequence'] = max(self.order_id.order_line.mapped('sequence'), default=0) + 1
```

Reply:
ok made the changes and here's new PR 
https://github.com/odoo/odoo/pull/168421